buildkite/src/Command/DockerArtifact.dhall

```diff
@@ -10,19 +10,25 @@ let Cmd = ../Lib/Cmds.dhall
 
 let DockerLogin = ../Command/DockerLogin/Type.dhall
 
-let commands : List Cmd.Type =
+
+let defaultArtifactStep = { name = "Artifact", key = "build-artifact" }
+
+let generateStep = \(deps : List Command.TaggedKey.Type) ->
+    -- assume head or first dependency specified represents the primary artifact dependency step
+    let stepScope = "--step ${(Prelude.Optional.default Command.TaggedKey.Type defaultArtifactStep (List/head Command.TaggedKey.Type deps)).name}"
```

Comment:
Setting build-artifact as default sounds good but then the GitTriggeredDockerRelease job needs to specify the proper TaggedKey explicitly

Reply:
Cool and makes sense. But yea afaict all uses of this job (Artifact.dhall Job and GithubTriggered... job) currently explicitly declare dependencies.

Reply:
No doubt and it does!